packages/app-store/zohocrm/lib/CrmService.ts

```diff
@@ -59,6 +59,8 @@ export default class ZohoCrmCrmService implements CRM {
   }
 
   async createContacts(contactsToCreate: ContactCreateInput[]) {
+    const auth = await this.auth;
+    await auth.getToken();
```

Comment:
These auth functions used to live in their parent methods. Since we're making these public need to include these.